packages/gestalt/src/Typeahead.js

```diff
@@ -0,0 +1,351 @@
+// @flow strict
+import React, { useState, useRef } from 'react';
+import PropTypes from 'prop-types';
+import InputField from './TypeaheadInputField.js';
+import Option from './TypeaheadOption.js';
+import Box from './Box.js';
+import Text from './Text.js';
+import Flyout from './Flyout.js';
+import Layer from './Layer.js';
+
+type TextColors =
+  | 'green'
+  | 'pine'
+  | 'olive'
+  | 'blue'
+  | 'navy'
+  | 'midnight'
+  | 'purple'
+  | 'orchid'
+  | 'eggplant'
+  | 'maroon'
+  | 'watermelon'
+  | 'orange'
+  | 'darkGray'
+  | 'gray'
+  | 'lightGray'
+  | 'red'
+  | 'white';
+
+type OptionObject = {
+  label: string,
+  value: string,
+};
+
+type Props = {|
+  accessibilityLabel: string,
+  id: string,
+  onChange: ({ event: SyntheticInputEvent<>, value: string }) => void,
+  data: Array<{
+    label: string,
+    value: string,
+  }>,
+  placeholder?: string,
+  size?: 'md' | 'lg',
+  value?: ?string,
+  searchField?: string,
+  onBlur?: () => void,
+  onChange?: ({
+    event: SyntheticEvent<HTMLInputElement>,
+    value: string,
+  }) => void => void,
+  onFocus?: () => void,
+  onSelect?: OptionObject => void,
+  value?: string,
```

Comment:
@simkessy `value` seems duplicated in this list, I also don't see it in the docs.

Could we make sure the docs + these flow types are in sync?